packages/insomnia/src/ui/components/error-boundary.tsx

```diff
@@ -65,8 +64,7 @@ class SingleErrorBoundary extends PureComponent<Props, State> {
           // @ts-expect-error -- TSCONVERSION
           message: (
             <p>
-              Failed to render {componentName}. Please send the following error to{' '}
-              <Mailto email="support@insomnia.rest" subject="Error Report" body={error.stack} />.
+              Failed to render {componentName}. Please report the error on our Github Issues: https://github.com/Kong/insomnia/issues
```

Comment:
Nit: We can add a link to GitHub issues:
```suggestion
              Failed to render {componentName}. Please report the error on our Github Issues: <a href="https://github.com/Kong/insomnia/issues">https://github.com/Kong/insomnia/issues</a>
```